recipes/rdkit/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "rdkit" %}
+{% set version = "Release_2017_09" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/rdkit/rdkit/archive/{{ version }}.tar.gz
+  sha256: f2a360f515968e79845e6257be9fee85809ce40705d5b383598de3747dbfbe6f
+  patches:
+    - rdpaths.patch
+    - rdconfig.patch  # [win]
+
+build:
+  number: 0
+
+requirements:
+  build:
```

Comment:
The most up-to-date of the `meta.yaml` that we use is here:
https://github.com/rdkit/conda-rdkit/blob/development/rdkit/meta.yaml

I see that there are some differences for conda-forge, but you'll certainly want to include cairo and pkg-config